modules/bvl_feedback/ajax/react_get_bvl_threads.php

```diff
@@ -19,32 +21,34 @@
 );
 
 require_once __DIR__ . "/../../../vendor/autoload.php";
-require_once "NDB_Client.class.inc";
 
-$user     =& User::singleton();
-$username = $user->getUsername();
+$username = \User::singleton()->getUsername();
+
 
 if (isset($_POST['candID']) && !(isset($_POST['sessionID']))) {
-    $feedbackThread =& NDB_BVL_Feedback::Singleton($username, $_POST['candID']);
+    $candID         = new CandID($_POST['candID']);
+    $feedbackThread =& \NDB_BVL_Feedback::Singleton($username, $candID);
 } elseif (isset($_POST['candID']) && isset($_POST['sessionID'])
     && !(isset($_POST['commentID']))
 ) {
+    $candID         = new CandID($_POST['candID']);
     $feedbackThread =&
-         NDB_BVL_Feedback::Singleton(
-             $username,
-             $_POST['candID'],
-             $_POST['sessionID']
-         );
+        \NDB_BVL_Feedback::Singleton(
+            $username,
+            $candID,
```

Comment:
Why isn't $username and $candID using $_Post to get it's information?

Reply:
`$candID` is populated by `$_POST` on lines 29 and 34.

In general it's not a good idea (at least by our team) to access PHP superglobals such as `$_POST` directly. We're trying to move away from it.